app/controllers/two_factor_authentication/otp_verification_controller.rb

```diff
@@ -3,17 +3,25 @@ class OtpVerificationController < DeviseController
     include TwoFactorAuthenticatable
 
     def show
-      @phone_number = decorated_user.masked_two_factor_phone_number
-      @code_value = current_user.direct_otp if FeatureManagement.prefill_otp_codes?
-      @delivery_method = params[:delivery_method]
+      assign_variables_for_otp_verification_show_view
     end
 
     def create
-      if current_user.authenticate_direct_otp(params[:code].strip)
+      result = OtpVerificationForm.new(current_user, form_params[:code].strip).submit
+
```

Comment:
that makes sense. I guess my assumption is that both new lines in a method are doing different things, but am happy to keep as-is and see if I catch onto this style